scripts/globals/job_utils/dancer.lua

```diff
@@ -55,7 +72,7 @@ local function setFinishingMoves(player, numMoves)
     numMoves              = math.min(numMoves, getMaxFinishingMoves(player))
 
     if finishingEffect then
-        if numMoves == 0 then
+        if numMoves <= 0 then
```

Comment:
We should never be purposefully going below 0 moves here, and if so, that might mask a greater problem

Reply:
I see now, but I don't like it.  This encourages bad behavior from other functions assuming that the set will handle it.  I would prefer additional logging happen here if it is a negative value, and still handle safety in the calling function as well.

Reply:
I reverted this change. The solution was easier than expected.